autoload.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
+// @TODO rename this file to bootstrap.php to better reflect its function
```

Comment:
Plan on doing it still in this PR or in a next release?

Reply:
next major